### PR TITLE
Doc - Support for Infrequent Tier SLOs.

### PR DESCRIPTION
**Pull request by hganapathy**
Created a doc for support of SLOs on Infrequent Tier. This doc should not be searchable
Also, removed Beta tag on Infrequent Tier Dashboard support page.

## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/manage/partitions-data-tiers/infrequent-tier-slo-support.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  <meta name="robots" content="noindex" />
+</head>
+
+Previously, SLOs were only supported in Sumo Logic's Continuous data tier. This support enabled you to configure SLOs on Infrequent tier logs to increase the monitoring coverage for reliability management.
+
+## Sample Queries
+
+While defining SLOs for the Infrequent tier logs, you can use the following sample queries.
+
+* Use `_dataTier=Infrequent` in your query scope. For example:
```
Comment (JainM6) [inline docs/manage/partitions-data-tiers/infrequent-tier-slo-support.md:19]: 1. Use `_dataTier=Infrequent` in your query scope. This ensures that your query only searches for data in the Infrequent tier. For example:

Comment (JainM6) [inline docs/manage/partitions-data-tiers/infrequent-tier-slo-support.md:19]:
```suggestion
* Use `_dataTier=Infrequent` in your query scope. This ensures that your query only searches for data in the Infrequent tier. For example:
```

File: docs/manage/partitions-data-tiers/infrequent-tier-slo-support.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  ```sql
+  _dataTier=Infrequent _sourceCategory=APIService | parse "Status: *" as status_code | where status_code >= 500
+  ```
+* Or select data in the Infrequent tier by specifying the Partition that contains it. For example:
```
Comment (JainM6) [inline docs/manage/partitions-data-tiers/infrequent-tier-slo-support.md:23]:
```suggestion
* Or, use `_index` parameter to specify the Partition that contains the data you want to access. For example:
```

File: docs/manage/partitions-data-tiers/infrequent-tier-slo-support.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  ```sql
+  _index=some_infrequent_index
+  ```
+  Or, run the following query:
```
Comment (JainM6) [inline docs/manage/partitions-data-tiers/infrequent-tier-slo-support.md:27]:
```suggestion
  Or, you can also search across multiple indices by using the `OR` operator. For example:
```

File: docs/manage/partitions-data-tiers/infrequent-tier-slo-support.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  ```sql
+  _index=some_infrequent_index OR _index=some_continuous_index
+  ```
+
+## Infrequent Dashboard Usage
+
+Sumo Logic provides a dashboard you can use to monitor the Infrequent tier consumption by SLOs.
```
Comment (JainM6) [inline docs/manage/partitions-data-tiers/infrequent-tier-slo-support.md:34]:
```suggestion
Sumo Logic provides a dashboard to monitor the Infrequent tier consumption by SLOs.
```

File: docs/manage/partitions-data-tiers/infrequent-tier-slo-support.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+<img src={useBaseUrl('img/partitions-data-tiers/infrequent-support-slo.png')} alt="dashboard" />
+
+The dashboard presents the following information for the currently selected time range:
+* **Avg Credits Consumed/Day**. The daily average number of credits consumed over the last 30 days.
+* **Total Credits Consumed**. The total credits consumed in the last 30 days.
+* **Top 50 Expensive Queries**. The top 50 expensive queries set up on Dashboards to run against the Infrequent tier.
+
+You can download the JSON for the dashboard [here](https://sumologic-app-data.s3.amazonaws.com/Infrequent_SLO.json). For information about importing dashboards and other content see [Import Content in the Library](/docs/get-started/library#import-content).
```
Comment (JainM6) [inline docs/manage/partitions-data-tiers/infrequent-tier-slo-support.md:43]:
```suggestion
You can download the JSON for the dashboard [here](https://sumologic-app-data.s3.amazonaws.com/Infrequent_SLO.json). For information about importing dashboards and other content, see [Import Content in the Library](/docs/get-started/library#import-content).
```

**Review by JainM6 — CHANGES_REQUESTED** [review]
Thanks Harrish for submitting the PR.  There are some minor formatting changes.

**Comment by kimsauce** [comment]
@hganapathy for the [infrequent-tier-dashboard-support](https://help.sumologic.com/docs/manage/partitions-data-tiers/infrequent-tier-dashboard-support) doc, can you confirm its production status? Looks like you removed the Beta tag, but you still want it hidden?

**Comment by hganapathy** [comment]
@Kim Pohas ***@***.***> this is already production ready but we
won't enable this for customers since this is on a limited availability
capacity. We don't want to term these as beta because scan charges incurred
by enabling these features will be charged to customers. But at the same
time, since its LA, we don't want it publicly visible.

**Comment by kimsauce** [comment]
> @kim this is already production ready but we won't enable this for customers since this is on a limited availability capacity. We don't want to term these as beta because scan charges incurred by enabling these features will be charged to customers. But at the same time, since its LA, we don't want it publicly visible.

@hganapathy Thank you for clarifying! Would it be ok if we added an LA note? Something like this?

_This feature is available on a Limited Availability (LA) basis. This means that while the feature is production-ready and fully supported, implementation is done by customer request only. If you'd like this enabled in your environment, contact your Sumo Logic account representative. The GA release of this feature will be announced in our [Release Notes](/release-notes-service)._

**Review by kimsauce — COMMENTED** [review]
Question re: LA

**Comment by hganapathy** [comment]
I think we shouldn't add any such note at this point. I should have given a
bit more context. Our goal with Data Tiers is to drive towards the Data
Tiers v2 (Flex) world and move away from Continuous/Infrequent/Frequent
tier world. However, this move is going to take time and we expect that
customers will still want a cost efficient way to manage their budget (ie)
send more logs into Infrequent tier.

However, since Infrequent previously did not support functions like
Dashboard, Monitors etc, there was a potential churn risk for customers who
opt to move their data to Infrequent because they don't derive enough
value. Hence, we will start supporting these features on a case by case
basis. Meaning, we will only activate these features for customers who are
a churn risk and that will be decided by a committee.

These features will never become GA and we will never proactively let any
customer know that these features are available. Hence the limited
availability and need to hide the pages. Happy to jump on a call and
explain.

**Review by kimsauce — APPROVED** [review]

**Pull request merged**